src/js/providers/default.js

```diff
@@ -94,9 +94,10 @@ const DefaultProvider = {
     },
 
     sendMediaType: function(levels) {
-        var type = levels[0].type;
-        var isAudioFile = (type === 'oga' || type === 'aac' || type === 'mp3' ||
-            type === 'mpeg' || type === 'vorbis');
+        const level = levels[0];
+        const type = level.type;
+        const isAudioFile = (type === 'oga' || type === 'aac' || type === 'mp3' ||
+            type === 'mpeg' || type === 'vorbis' || level.mimeType === 'audio/mp4');
```

Comment:
Do `level.mimeType.indexOf('audio/') === 0` to support all mime-types starting with 'audio/' .

Reply:
Re source.js:  We could consider adding `mediaType` as a property of source in a future release, but should not change how `source.type` is set, because it could impact existing integrations with our API.